Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -716,8 +716,12 @@ static void logProcessPoolState(const WebProcessPool& pool)
         "com.apple.WebKit.showMemoryCache"_s,
         "com.apple.WebKit.showPaintOrderTree"_s,
         "com.apple.WebKit.showRenderTree"_s,
+        "com.apple.CFPreferences._domainsChangedExternally"_s,
```

Comment:
Do we need to update web process's sandbox profile to allow these for darwin-notification-post?

Reply:
Ah, that is a good catch, fixed in latest patch!

Thanks for reviewing!